docs/proposals/horizontal-pod-autoscaler.md

```diff
@@ -218,10 +221,6 @@ Such approach has two benefits:
 
 * Autoscaler avoids thrashing, i.e.: prevents rapid execution of conflicting decision if the load is not stable.
 
-
```

Comment:
Because it is above: "Starting and stopping pods may introduce noise to the metric..."